stacks/cli/src/commands/delete.ts

```diff
@@ -0,0 +1,67 @@
+import {ActionParameters, CaporalValidator, Command, CreateCommandParameters} from "@caporal/core";
+import * as fs from "fs";
+import * as path from "path";
+import * as YAML from "yaml";
+import {machinery} from "../machinery";
+import {formatFailureStatus, isFailureStatus} from "../status";
+
+async function del({options}: ActionParameters) {
```

Comment:
`remove` is more understandable than `del` I think.

Reply:
replaced with `_delete`. I could not use delete cause it is reserved.

Reply:
I suggested `remove` word but it is okay